Pipfile

```diff
@@ -17,6 +17,7 @@ aio_pika = "*"
 aiocron = "*"
 prometheus_client = "*"
 yoyo-migrations = "*"
+pamqp = "==2.3.0"
```

Comment:
What is pamqp being used for? I'm not seeing it imported anywhere

Reply:
It's used in the message queue service, it has been there before, but the update of the packages caused an increase in the major version of pamqp, which broke the code, so I pinned it on the working version.